.github/SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+Thank you for helping us keep our software safe and secure. We value the security community and the important work you do. This document outlines our approach to handling security vulnerabilities.
+
+## Supported Versions
+
+## Reporting a Vulnerability
+
+**Please responsibly disclose any potential security vulnerabilities by mailing the following address:**
```

Comment:
If you believe to have discsovered a potential vulnerabiltiy relating to Form3's systems, please responsibly disclose your findings to our team using the the following mailbox: